internal/cobraprompt/prompt.go

```diff
@@ -146,18 +151,27 @@ func (co CobraPrompt) Run(ctx context.Context, root *cobra.Command, cnfg *hazelc
 			root, _ = rootcmd.New(cnfg)
 			prepareRootCmdForPrompt(co, root)
 			root.SetArgs(promptArgs)
+			root.SetFlagErrorFunc(func(_ *cobra.Command, err error) error {
+				return hzcerror.FlagError(err)
+			})
 			if err := root.ExecuteContext(ctx); err != nil {
 				if errors.Is(err, ErrExit) {
 					exitPromptSafely()
 					return
 				}
+				if err.Error() == `required flag(s) "name" not set` {
+					// todo make this applicable for all data types
+					err = fmt.Errorf(`%s. Add it or consider "map use <name>"`, err.Error())
```

Comment:
Would wrapping the error work better instead of inserting the error message?

Reply:
Thanks, fixed at https://github.com/hazelcast/hazelcast-commandline-client/pull/61/commits/9cb4ca319bb38991b8f1d9e773f2bf329b24d6e6